add_to_app/books/flutter_module_books/lib/api.dart

```diff
@@ -94,6 +94,7 @@ abstract class FlutterBookApi {
       if (api == null) {
         channel.setMessageHandler(null);
       } else {
+        // ignore: avoid_types_on_closure_parameters
         channel.setMessageHandler((Object? message) async {
```

Comment:
Is the type necessary here?

Reply:
I don't recall, but I usually leave types in when things break on removal of said type declaration